src/main/java/uk/gov/hmcts/ccd/security/filters/LoggingFilter.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.ccd.security.filters;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.filter.OncePerRequestFilter;
+import uk.gov.hmcts.ccd.data.SecurityUtils;
+import uk.gov.hmcts.reform.idam.client.models.UserInfo;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.regex.Pattern;
+
+@Slf4j
+public class LoggingFilter extends OncePerRequestFilter {
+
+    private static final String LOG_MESSAGE_TEMPLATE = "[LOG FILTER] Attempting to serve request %s %s for user with IDAM roles %s";
```

Comment:
doesn't LOG4J add the class name (`[LOG FILTER]`) already by default?

Reply:
Initially added to enable searching by a consistent term on AI (as L&A does), but good point, probably not much point as can still search by logger name. Removing.